DESCRIPTION

```diff
@@ -39,8 +39,9 @@ Authors@R: c(person("Dirk", "Eddelbuettel", role = c("aut", "cre"), email = "edd
              person("Winston", "Chang", role="ctb", comment = c(ORCID = "0000-0002-1576-2126")),
              person("Dean", "Attali", role="ctb", comment = c(ORCID = "0000-0002-5645-3493")),
              person("Michael", "Chirico", role="ctb", comment = c(ORCID = "0000-0003-0787-087X")),
-             person("Kevin", "Ushey", role="ctb"))
-Version: 0.6.37
+             person("Kevin", "Ushey", role="ctb"),
+             person("Carl", "Pearson", role="ctb", comment = c(ORCID = "0000-0003-0701-7860")))
+Version: 0.6.38
```

Comment:
Please making 0.6.37.1.   It's a bit of my personal tick to leave the determination of what the release is to the maintainer (me) but like you I also _much_ insist of making in-flight versions different from released one.

Been using four-digit 'major.minor.patch.micro' for that since forever.  Works well.

Reply:
Done.